src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js

```diff
@@ -17,14 +17,13 @@ import useItemAcquisition from './useItemAcquisition';
 
 const ItemAcquisition = ({ accordionId, itemId }) => {
   const stripes = useStripes();
-  const { isLoading, itemAcquisition = {} } = useItemAcquisition(itemId);
+  const { isLoading, itemAcquisition = {}, isCentralTenantAcq } = useItemAcquisition(itemId);
```

Comment:
Pedantic: consider parity in how this boolean is named. Everywhere else it's `Acquisition` so why abbreviate it `Acq` here? Consistency makes it more obvious that these things are related and reduces the cognitive load.